notebooks/Getting_started_with_Forecast.ipynb

```diff
@@ -464,8 +434,32 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "createPredictorResponse=forecast.create_predictor(RecipeName='forecast_MQRNN',DatasetGroupName= datasetGroupName ,PredictorName=predictorName, \n",
-    "  ForecastHorizon = forecastHorizon)"
+    "create_predictor_response=forecast.create_predictor(PredictorName=predictorName, \n",
+    "                                                  AlgorithmArn=algorithmArn,\n",
+    "                                                  ForecastHorizon=forecastHorizon,\n",
+    "                                                  PerformAutoML= False,\n",
+    "                                                  PerformHPO=False,\n",
+    "                                                  EvaluationParameters= {\"NumberOfBacktestWindows\": 1, \n",
+    "                                                                         \"BackTestWindowOffset\": 24}, \n",
+    "                                                  InputDataConfig= {\"DatasetGroupArn\": datasetGroupArn},\n",
+    "                                                  FeaturizationConfig= {\"ForecastFrequency\": \"H\", \n",
+    "                                                                        \"TrainingSubsampleRatio\": 1, \n",
```

Comment:
Should we consider omitting out FeaturizationConfig?

Reply:
Keeping in this example and the autoML one. Removed in bikeshare example to show usage.

Reply:
Scratch that Featurization Config is required.